packages/core/src/state-management/__tests__/anchor-resuming.test.ts

```diff
@@ -0,0 +1,93 @@
+import { expect, jest } from '@jest/globals'
+import { createCeramic } from '../../__tests__/create-ceramic.js'
+import { AnchorStatus, IpfsApi, SyncOptions, TestUtils } from '@ceramicnetwork/common'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+import tmp from 'tmp-promise'
+import { createIPFS } from '@ceramicnetwork/ipfs-daemon'
+import all from 'it-all'
+
+jest.setTimeout(10000)
+
+let ipfs: IpfsApi
+let stateStoreDirectoryName: string
+
+beforeAll(async () => {
+  ipfs = await createIPFS()
+})
+
+afterAll(async () => {
+  await ipfs.stop()
+})
+
+beforeEach(async () => {
+  stateStoreDirectoryName = await tmp.tmpName()
+})
+
+afterEach(async () => {
+  jest.resetAllMocks()
+})
+
+test('resume anchors from AnchorRequestStore', async () => {
+  const numberOfStreams = 3
+
+  const ceramic = await createCeramic(ipfs, {
+    stateStoreDirectory: stateStoreDirectoryName,
+    enableLoop: false,
```

Comment:
this looks weird at this level to just see `enableLoop` without context.  Maybe can we call it `enableAnchorPollingLoop`?